src/main/resources/application.yml

```diff
@@ -1,16 +1,24 @@
 scheduler:
   emailreader:
+    name: fetch_emails
     cron: 0 */5 * * * *
     shedlock-lock-at-most-for: PT10M
+    maximum-execution-time: PT2M
```

Comment:
We should make sure cron, schedlock and maximum align with each other.  cron >= shedlock >= maximum-exec-time

Reply:
Maybe it is enough to just set cron intervall == maximum exec-time (to get at warning), and let schedlock be twice the max-exec-time (to avoid double execution)...